.travis.yml

```diff
@@ -46,14 +44,18 @@ jobs:
     - ACCEPT_EULA=Y
     - MSSQL_PID='evaluation'
     before_install:
-    - sudo apt-get install -y mssql-tools mssql-server
+    - sudo apt-get install -y --no-install-recommends mssql-tools mssql-server dotnet-runtime-2.2 dotnet-sdk-3.1
     - sudo /opt/mssql/bin/mssql-conf -n setup accept-eula
     install:
     - ( cd ./lib/java && ./installDat.sh && cd - )
-    - wget https://github.com/HicServices/RDMP/releases/download/v3.2.1/rdmp-cli-linux-x64.zip
-    # RDMP 3.2.1 was packaged using a ZIP tool with a bug Microsoft fixed in .Net 4.6.1; excluding Chinese language support file and keyword help avoids unzip being affected by the bug: https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/mitigation-ziparchiveentry-fullname-path-separator?redirectedfrom=MSDN
-    - unzip -d rdmp-cli rdmp-cli-linux-x64.zip -x "Curation*" "zh-*"
-    - chmod +x rdmp-cli/rdmp
+    - |
+      if [ ! -d rdmp-cli ]
```

Comment:
Should this check/install to $HOME/rdmp-cli (which is cached)?

Reply:
This seems to have run in the last few builds, even though it should be cached by now